webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -171,9 +223,45 @@ const useModalModelCreate = ({ onClose }: Props = {}): ModalHooks => {
             <Form.Item
               label="Model name"
               name="modelName"
-              rules={[{ message: 'Model name is required ', required: true }]}>
+              required
+              requiredMessage="Model name is required ">
               <Input onChange={handleNameChange} />
             </Form.Item>
+            {isRbacEnabled && (
```

Comment:
https://determinedai.atlassian.net/browse/WEB-805 reads as though models should always be paired to workspace. And `rbac` comes into play to decide which workspaces are shown to a user. 

Can you confirm that we only want to add a `workspace` if rbac is enabled? Which I am guessing is not the case. 

If it is the case this is good as is, otherwise changes will be needed.

Reply:
Workspace dropdown should display even if rbad is off.